lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -548,6 +559,110 @@ func (c *ContainerProxy) createNewAttachClientWithTimeouts(connectTimeout, respo
 	return plClient, transport
 }
 
+func (c *ContainerProxy) Resize(vc *viccontainer.VicContainer, height, width int32) error {
+	defer trace.End(trace.Begin(vc.ContainerID))
+
+	if c.client == nil {
+		return derr.NewErrorWithStatusCode(fmt.Errorf("ContainerProxy failed to create a portlayer client"),
```

Comment:
I'd rather wait on this.  The InternalServerError message is wrong.  We need to go through and clean those up again.  This isn't an internal error from portlayer.  We need finer grain error messages in errors.go.